data/css/modules/card/_legacyPolaroid.scss

```diff
@@ -7,7 +7,7 @@
         padding: 4px 0px;
         color: $accent-dark-color;
         font-family: $title-font, sans-serif;
-        font-size: 0.77em;
+        font-size: 0.67em;
```

Comment:
Fair enough, though of course this issue may still manifest itself on other apps while we add more localizations.